ui/src/components/workflow/WorkflowMetaDetails.js

```diff
@@ -60,10 +61,8 @@ class WorkflowMetaDetails extends Component {
           <Tab eventKey={1} title="Diagram">
             <div><WorkflowMetaDia meta={wf} tasks={[]}/></div>
           </Tab>
-          <Tab eventKey={2} title="JSON">
-            <div><pre>
-              {JSON.stringify(this.state.workflowMeta, null, 2)};
-          </pre></div>
+          <Tab ref={this.JSONTab} eventKey={2} title="JSON">
+            <JSONTab wfs={JSON.stringify(this.state.workflowMeta, null, 2)} name={this.state.name} version={this.state.version}></JSONTab>
```

Comment:
We should encapsulate this line in div, also no need for enclosing tag if we have no component children.